### PR TITLE
Added script to push docker images to a specified registry

### PR DESCRIPTION
**Pull request by iceglow**
Don't know if this is at all desirable, but I added a script to push images to a specified docker registry. We use it at my company, just close it if it is not interesting.

I tried to copy the script structure used for build & run.

### CODE REVIEW

**Comment by chamilad** [comment]
@iceglow This looks cool. Thanks!

File: common/scripts/docker-push.sh [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        docker tag "${image_name_and_version}" "${registry_url}/${image_name_and_version}"
+        docker push "${registry_url}/${image_name_and_version}"
+    } || {
+        echoError "Couldn't push image ${image_name_and_version} to registry ${registry_url}"
+        exit 1
+    }
+
+    product_name_in_uppercase=`echo ${product_name} | tr '[:lower:]' '[:upper:]'`
+    echoSuccess "${product_name_in_uppercase} ${profile} pushed image: [image] ${image_name_and_version} to [registry] ${registry_url}"
+    sleep 1
```
Comment (ravihansa3000) [inline common/scripts/docker-push.sh]: what's the need for a sleep here? Just asking :)

Comment (iceglow) [inline common/scripts/docker-push.sh]: :D remnant of the docker-run script I copied the base for that section from ;)

**Comment by ravihansa3000** [comment]
@iceglow Thank you!

**Pull request merged**